5.4.x testnet node release (#820)
Concordium/concordium.github.io: source/mainnet/net/resources/release-notes.rst

   To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.

June 1, 2023

Concordium Node 5.4.1
---------------------

Concordium node version 5.4.1 contains the following features and bug fixes:

- Enable CORS support in grpc-web. This only applies when grpc-web is enabled.

- Fixed a security issue.

- Support using block height as block identifiers in gRPC v2 API.

- Extend gRPC v2 API call ``GetBlockInfo`` with the protocol version of the block.

> - Do not use peer buckets when running as a normal node (not as a bootstrapper).

Reviewer: ```suggestion
- Do not keep a historical list of peers when running as a normal node.
```

Concordium Client 5.2.0
-----------------------

Version 5.2.0 contains the following features and bug fixes:

> - Fixed a bug that caused another code-leftover to be displayed in the finalization proof line of the gas rewards section of the ``consensus show-chain-parameters`` output. This is only relevant when the queried chain runs protocol version 6.

Reviewer: ```suggestion
- Fix a bug in display of ``consensus show-chain-parameters`` output for protocol version 6.
```

> - Added ``raw GetBlockTransactionEvents`` that prints the list of transaction outcomes in a given block.

Reviewer: ```suggestion
- Add ``raw GetBlockTransactionEvents`` that prints the list of transaction outcomes in a given block.
```
